esp-hal/src/rtc_cntl/mod.rs

```diff
@@ -113,7 +113,7 @@
 pub use self::rtc::SocResetReason;
 #[cfg(not(esp32))]
 use crate::efuse::Efuse;
-#[cfg(any(esp32, esp32s2, esp32s3, esp32c3, esp32c6, esp32c2))]
+#[cfg(any(esp32, esp32s2, esp32s3, esp32c3, esp32c6, esp32c2, esp32h2))]
```

Comment:
Nit: I guess we can remove those (on other places as well) as all currently supported chips are included?
```suggestion
```

Reply:
Ideally we'd define sleep support in metadata, cleaning this up isn't important until we can do it. We will likely add more devices first anyway, and then this `cfg` will need to be reintroduced.